docs/tutorials/subscription-platform.md

```diff
@@ -71,6 +71,8 @@ Product Names are the canonical displayed name shown in Sub Plat UI. In some cas
 | emailIconURL                              | Optional. Image URL for product icon in email content. This must be a URL to the FxA CDN at <https://accounts-static.cdn.mozilla.net>. |
 | appStoreLink                              | Optional. The App store download URL for the product. |
 | playStoreLink                             | Optional. The google play store download URL for the product. |
+| newsletterSlug                            | Optional. Comma seperated string array of slugs to send if a customer opts to signup for the newsletter. Valid values are `mozilla-accounts`, `security-privacy-news`, `hubs`, `mdnplus`, which can be combined into a comma separated string. Defaults to `mozilla-accounts`. |
+| newsletterLabelTextCode                   | Optional. A code used to determine which of the predefined labels to add to the newsletter checkbox. Valid values are `snp`, `hubs`, `mndplus`. If no code is provided, the default label text is displayed. |
```

Comment:
Nit: do we need to clarify what the default label text would be?

Reply:
I don't think so. I could add a link to Storybook which will have an example of all of the strings